src/test/java/com/wikia/webdriver/testcases/portableinfoboxtests/PortableInfoboxTests.java

```diff
@@ -46,15 +45,18 @@
  * TC18: Additional <div> wrappers from title, header and image HTML are removed
  * TC19: Verify that any of the tags which do not have a value won't appear
  */
+@Test(groups = "PortableInfoboxTests")
 public class PortableInfoboxTests extends NewTestTemplate {
 
   private final static Logger LOGGER = Logger.getLogger(PortableInfoboxTests.class.getName());
 
-  @Test(groups = {"PortableInfoboxTests", "PortableInfoboxTests_001"})
+  // TODO: compress this test
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)